src/ContaoCommunityAlliance/DcGeneral/Contao/Subscriber/FallbackResetSubscriber.php

```diff
@@ -113,6 +114,11 @@ private function handleFallback(AbstractModelAwareEvent $event)
                     $dataProvider->resetFallback($propertyName);
                 }
 
+                // If a fallback already exists, skip the empty one.
```

Comment:
Could be phrased better `If value is empty, no need to reset the fallback.`

Reply:
done